### DISCUSSION TITLE
one_off campaign by email and widget

### DISCUSSION THREAD

**Discussion by MisterRails** [chatwoot/chatwoot#4470] [Feature Requests]
Hi,
We actually can use campaigns only by triggering a message to everyone (ongoing = based on an url and a duration) or a SMS campaign (one_off = only with sms).

In y case, i really could use to activate certain people based on some labels (or custom attribute) by email, doubled with a message in the widget chatbox like an ongoing campaign.

**Comment by sojan-official** [comment]
ref: https://github.com/chatwoot/chatwoot/issues/4380
